src/averageAges.js

```diff
@@ -15,11 +15,14 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const menFilter = century ? people.filter(person =>
+    Math.ceil(person.died / 100) === century && person.sex === 'm')
```

Comment:
Consider moving the complex condition inside the filter into a variable for better readability. For example, you could define a variable like `isCenturyMan` that encapsulates the logic.